spec/controllers/queue_items_controller_spec.rb

```diff
@@ -0,0 +1,100 @@
+require 'spec_helper'
+
+describe QueueItemsController do
+  describe 'GET index' do
+    it "assigns @queue_items to the queue items of the current user" do
+      alice = Fabricate(:user)
+      session[:user_id] = alice.id
+      queue_item1 = Fabricate(:queue_item, video_id: 1, user: alice )
+      queue_item2 = Fabricate(:queue_item, video_id: 2, user: alice )
```

Comment:
I am not seeing in this spec where do you fabricate video objects. You should not create a queue item if the video does not exist.